Remotion/Core/Reflection.UnitTests/ReflectionExtensionsTest.cs

```diff
@@ -116,7 +116,7 @@ public void ConvertToRuntimeType_WithOtherITypeInformation_ThrowsInvalidOperatio
           Throws.InvalidOperationException.And.Message.EqualTo (
               string.Format (
                   "The type 'TheName' cannot be converted to a runtime type because no conversion is registered for '{0}'.",
-                  typeInformation.GetType().Name)));
+                  typeInformation.GetType().FullName)));

```

Comment:
What's the motivation for this change? Is Name not specific enough?

Reply:
The actual exception message changed, probably with the Castle upgrade in CoreRhinoMocks